CHANGELOG.mkd

```diff
@@ -3,10 +3,23 @@ CHANGELOG
 
 Unreleased
 ----
-- Add exec environment source type. The exec source type allows for the implementation of external environment sources [#1042](https://github.com/puppetlabs/r10k/pull/1042)
+
+
+3.5.0
+-----
+
+- Add exec environment source type. The exec source type allows for the
+  implementation of external environment sources
+  [#1042](https://github.com/puppetlabs/r10k/pull/1042).
+- Improve atomicity of .r10k-deploy.json writes. Fixes
+  [#813](https://github.com/puppetlabs/r10k/issues/813)
+- Fix option checks from newest CRI bump. See
```

Comment:
I didn't really think this warranted a changelog entry, since it doesn't change functionality..

Reply:
My impression was that it fixed a regression introduced in the last release + latest CRI. Like I mentioned in my other comment, I couldn't really follow the ticket.

Reply:
I'll just remove this and kill two birds with one stone.

Reply:
I think the latest CRI _fixes_ the regression. We had previously pinned back to avoid it. @pcarlisle Do you know if we're in a broken state right now, with cri 2.15.10 (latest) but with your options fixes not-quite-released?